.github/ISSUE_TEMPLATE/1-bug-report.yml

```diff
@@ -35,6 +35,30 @@ body:
       placeholder: ex. 7.6.0
     validations:
       required: true
+  - type: input
+    id: node-version
+    attributes:
+      label: Node version
+      description: What version of node.js are you using to run Cypress?
+      placeholder: ex. v16.16.0
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: Operating System
+      description: What operating system is Cypress running on when you are encountering your problem?  The more specific the better  ie macOS 12.4 or Windows 10.0.19044.1889
+      placeholder: ex. macOS 12.4
+    validations:
+      required: true
+  - type: textarea
```

Comment:
I'm not a fan of this input. In most cases we care more about a reproduction than we do about the debug logs; They're a heavy club of last resort when all else fails, and I don't want to make them required for opening an issue.

Reply:
I wouldn't go so far as to say they're a last resort, but I agree that they're not needed/helpful for every class of issue. We could leave this as an optional field, unless we think its presence will cause undue friction. 

If we do keep it, this link (https://docs.cypress.io/guides/references/troubleshooting#Print-DEBUG-logs) is probably more applicable. The current link is targeted towards debugging specs within the browser more-so than Cypress itself.

Reply:
I agree it is a heavier lift than most of the information.  But based on the tickets I have seen coming in over the last month or two, we need to raise the bar on expectations of the community when entering a ticket if for no other reason than we simply don't have the resources to triage in an effective way on a majority of issues entered.  By asking for this information up front we can more quickly dive deeper into issues without slow back and forth async communication where we are never sure if someone is going to ever get back to us (they often don't).  I am hoping with clearer details such as debug logs from the start we can get through more issues and ultimately have time to fix more issues with the limited resources we have as a consequence.

Reply:
I agree about raising the bar - but I'd rather do that by asking for reliable reproductions, rather than debug logs.

Perhaps instead, a required link to "Repository demonstrating the issue, or CI build Cypress team can run"? Something like that. I'd be fine with this as an optional field, but as a required one it feels like it's just adding friction to the process and not actually providing what we need - which is a way to reproduce issues and more quickly locate the heart of the problem.

Reply:
Debug logs put all the effort on us, to read through the logs and make sense of them, when what we actually care about is asking users to put in more effort.

Reply:
I like asking for it up front, but perhaps not requiring it. We could update the description to indicate that we may have to ask for it later on, though, and that it could delay investigating their issue.

Beyond that, though, I'm concerned that without the additional `render` attribute, these logs could make scrolling through the issue incredibly painful. It looks like this attribute will not expand the field to show the entire content (if I'm understanding the [textarea render attribute doc](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#attributes-1) correctly). Perhaps we add that attribute with a value of `Text` for language?

Reply:
@BlueWinds @rockhold I have set required to false based on general feedback.  

@rockhold good point on the rendering of the textarea for debug logs.  I have added render: shell based on some examples I could find online.  We can change to text if this parsing isn't optimal in everyday life.

@tbiethman thank for the more relevant debug log link.  I have updated the URL.

I also tried to go through and standardize wording on fields and punctuation but that is not my strong suit so if you all notice any other inconsistencies worth fixing let me know.